recipes/dev/lora_finetune_fsdp2.py

```diff
@@ -27,7 +27,7 @@
 from torchtune import config, modules, utils
 from torchtune.datasets import ConcatDataset
 from torchtune.modules.peft import LoRALinear
-from torchtune.modules.peft.peft_utils import (
+from torchtune.modules.peft._utils import (
```

Comment:
This is actually incorrect. Our recipes should really only be accessing public APIs (no underscore). Therefore, the proper way to do this would be to update the `__init__.py` file in `modules/peft` to import the APIs from `_utils.py`. Then, these import statements would look like:

```
from torchtune.modules.peft import LoRALinear, get_adapter_params, ...
```

Reply:
whoops. thank you